test/AtlasVerificationNonces.t.sol

```diff
@@ -3,15 +3,50 @@ pragma solidity 0.8.22;
 
 import "forge-std/Test.sol";
 
+import { NonceManager } from "src/contracts/atlas/NonceManager.sol";
```

Comment:
File names for tests for NonceManager do not match, they are hard to find. Suggest renaming tests from `AtlasVerificationNonces.t.sol` to `NonceManager.t.sol`.

Reply:
https://github.com/FastLane-Labs/atlas/pull/259/commits/a5528e1c7cf78a7878b44f1304070c6156f6b07c